### ISSUE TITLE
Fanging Issues - Spaces in Regex.

### ISSUE THREAD

**Issue by git-yogeshkumar** [ioc-fang/ioc-fanger#46]
@fhightower We are using this library for a long time, thanks for doing such awesome work, recently after the upgrades we are receiving cases that are increasing false positive detections for us, I wanted to raise a few concerns. The regex that are updated involves spaces to be considered. Here are few examples that are not producing correct results.

```
ioc_fanger.fang("div><div><br></div><div>hxxp://zeplin[.]atwebpages[.]com/inter[.]php</div><")
Output :  'div><div><br></div><divhttps://ziouo.aspages.com/outer.php</div><'

ioc_fanger.fang("applications. [To")
Ouput : 'applications.To'
```
In the first case we are missing out on the URL and in the second one we are generating a False Positive Domain.
Also Support for angle brackets ('<' , '>') in the near updates can be added cause there are cases where we have seen that defanged ioc's have angle brackets.

My team also wanted to know if there was a specific use case for which we brought up spaces in regex, if you can help with the above things that would be great.

**Comment by fhightower** [comment]
Thanks for the kind words @git-yogeshkumar! And thanks for bringing these concerns to my attention.

ioc-fanger is certainly wrong in each of those examples you provided and should be updated. I'll look into each of the two issues you noted and get back to you with more details soon.

**Comment by fhightower** [comment]
I've created #47 and #48 as derivatives from your comments. #48 is resolved with version 3.2.1 and I hope to have #47 resolved very soon.

**Comment by fhightower** [comment]
@git-yogeshkumar : #47 is resolved in version `3.2.2` (recently released).

Thanks again for raising this issue and please let me know if you have other questions!
